bin/verify-tag.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function log() {
+    echo "$1" 1>&2
+}
+
+function bail() {
+    log "$1"
+    exit 1
+}          
+
+function main() {
+    VERSION_TAG=$1
+    tag_arr=(${VERSION_TAG//-/ })
+    if [ "${#tag_arr[@]}" != 2 ]; then
+        bail "Tag must have a release sequence, <vYYYY.MM.DD-[0..n]>"
+    fi
+    todays_date=$(date +'v%Y.%m.%d')
```

Comment:
make this utc today as we have [here](https://github.com/replicatedhq/kURL/blob/bef801b49c7b36e17d7d6f7ad52972bd2a70b1c3/bin/tag-and-release.sh#L72)

```suggestion
    todays_date=$(date -u +'v%Y.%m.%d')
```